### ISSUE TITLE
New Term: Poly-hill sign

### ISSUE THREAD

**Issue by CorneliusDreyling** [obophenotype/human-phenotype-ontology#10327]
**Preferred term label:**
Poly-hill Sign

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
A typical physical examination in patients with muscular dystrophies and selective weakness and atrophy of the trapezius muscle and proximal deltoid muscle. The contour of the back shows unique 'hills' and drops. (PMID: 11994973)
DOI:[10.1136/practneurol-2015-001353]

**Parent term (use hpo.jax.org/app)**
 HP:0001446

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Clinical sign in muscular dystrophies like facioscapulohumeral dystrophy 

**Your nano-attribution (ORCID)**
0009-0004-9942-3886

**Comment by pnrobinson** [comment]
@CorneliusDreyling thanks!
Adding new term: 
- ID: HP:6000504
- label: Poly-hill sign
- parents: [Abnormality of the musculature of the upper limbs (HP:0001446)](https://hpo.jax.org/app/browse/term/HP:0001446)
- pmid: PMID:11994973
- def: A typical physical examination in patients with muscular dystrophies and selective weakness and atrophy of the trapezius muscle and proximal deltoid muscle. The contour of the back shows unique hills and drops.
- comment: 
- orcid: orcid:0009-0004-9942-3886
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10327
- synonyms:
